RAP/rap-statistics.qmd

```diff
@@ -574,20 +574,38 @@ Peer review is an important element of quality assuring our work. We often do it
 
 Prior to receiving code for peer review, the author should ensure that all code files are clean, commented appropriately and for larger projects should be held in a repo with an appropriate [README](#writing-a-readme-file) file.
 
-When peer reviewing code you should be consider the following questions - 
+When peer reviewing code you should consider the following questions - 
 
```

Comment:
All looks really good, I think near the beginning it would be good to link to the duck book peer review page but other than that all good!!!!

Reply:
Yeah makes sense! I'll add that in 😄